content/docs/command-reference/exp/remove.md

```diff
@@ -40,6 +40,8 @@ With `--queue`, the list of experiments awaiting execution is cleared instead.
 - `-g`, `--git-remote` - Name or URL of the Git remote to remove the experiment
   from
 
+- `--keep` - keep only the selected experiments and remove all others
```

Comment:
minor / stylistic:

how about this:

> changes the default behavior to the opposite for options like `-n`, `--rev`. For example `-n <num> --keep` _keeps_ experiments from the last `num` commits and _removes all_ other experiments.


does it change behavior for `--queue`? what is the behavior for `-A`?

Reply:
As of now, it raises an exception with `--queue` (InvalidArgumentError), until we decided what to do with `--queue`  as mentioned in [dvc - #10639](https://github.com/iterative/dvc/issues/10639)

As one might expect, `--keep -A` (or `--keep --all-commits`) keeps all experiments and removes nothing, basically just wasting a bit of energy, iops and cpu cycle. In terms of experiments and data, this has the same outcome as not running the command at all.